README.md

```diff
@@ -861,6 +861,17 @@ word-break:break-word
      </td>
      <td>Read</td>
    </tr>
+  <tr>
```

Comment:
This should be in the README-template only, not README

Reply:
Fixed